cmd/opentelemetry/app/defaultconfig/default_config.go

```diff
@@ -188,10 +199,14 @@ func receiverNames(receivers configmodels.Receivers) []string {
 	return names
 }
 
-func processorNames(processors configmodels.Processors) []string {
+// processorNames returns processor names that are present in both input parameters
```

Comment:
Why would the list of processors be different?

Reply:
the resource processor is enabled only when needed.